Classes/Indexer/Object/Transform/DateTransformer.php

```diff
@@ -36,6 +36,14 @@ public function getTargetMappingType()
      */
     public function transformByAnnotation($source, TransformAnnotation $annotation)
     {
+        if ($source === null) {
+            return null;
+        }
+
+        if (is_string($source)) {
```

Comment:
Is this really related to "allow nullable date" ?

Reply:
Nope, the pr is based on an older version probably. This is an updated pr @dfeyer 
https://github.com/Flowpack/Flowpack.ElasticSearch/pull/79